### PR TITLE
Feat/Pld-2 Configure Frontend Packages

### PR DESCRIPTION
**Pull request by VeloOFFICIAL**

### CODE REVIEW

File: apps/web/src/app/_nextThemes/Provider.tsx
Comment (masterHAWK99) [inline apps/web/src/app/_nextThemes/Provider.tsx:1]: Use kebab case for file names to keep it consistent

File: apps/web/src/app/_reactQuery/Provider.tsx
Comment (masterHAWK99) [inline apps/web/src/app/_reactQuery/Provider.tsx:1]: Use kebab case for file names to keep it consistent

File: apps/web/src/utils/httpClient.ts
Comment (masterHAWK99) [inline apps/web/src/utils/httpClient.ts:1]: Use kebab case for file names to keep it consistent

File: packages/ui/.storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  docs: {
+    autodocs: "tag",
+  },
+  webpackFinal: (config) => {
+    if (config && config.resolve) {
+      config.resolve.plugins = [new TsconfigPathsPlugin()];
+    }
+
+    return config;
+  },
```
Comment (masterHAWK99) [inline packages/ui/.storybook/main.ts:39]: You can use nextjs framework which provide support for absolute paths

File: packages/eslint-config/eslint-next.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:@tanstack/eslint-plugin-query/recommended",
+    "./eslint-base.json"
+  ],
+  "plugins": ["react", "react-hooks", "jsx-a11y", "@tanstack/query"],
+  "rules": {
+    "react/react-in-jsx-scope": "off"
+  }
```
Comment (masterHAWK99) [inline packages/eslint-config/eslint-next.json:14]: You can use next config which contains those

File: apps/web/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "@/*": ["./src/*"]
+    },
+    "skipLibCheck": true,
+    "strict": false,
+    "noEmit": true,
+    "incremental": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "resolveJsonModule": true,
+    "isolatedModules": true
```
Comment (masterHAWK99) [inline apps/web/tsconfig.json:14]: Most of these properties are inherited, so I think you can remove them from here

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "frontend",
```
Comment (PatrykKuniczak) [inline package.json:2]: should be
```suggestion
  "name": "privatelessonsdashboardfrontend",
```

**Pull request closed without merging**